dll/win32/browseui/shellbrowser.cpp

```diff
@@ -2585,12 +2662,14 @@ HRESULT STDMETHODCALLTYPE CShellBrowser::UpdateBackForwardState()
 
 HRESULT STDMETHODCALLTYPE CShellBrowser::SetFlags(DWORD dwFlags, DWORD dwFlagMask)
 {
-    return E_NOTIMPL;
+    m_BrowserSvcFlags = (m_BrowserSvcFlags & ~dwFlagMask) | (dwFlagMask & dwFlags);
```

Comment:
```suggestion
    m_BrowserSvcFlags = (m_BrowserSvcFlags & ~dwFlagMask) | (dwFlags & dwFlagMask);
```
it's obviously the same, but here the suggestion is to have the flags variables on which the mask acts, on the left (and the mask variable on the right)